src/organisms/organism.py

```diff
@@ -24,11 +25,14 @@ def update_lifetime(self):
     def draw(self, config, display):
         pygame.draw.circle(display, (255, 0, 0), (self.position.x * config.tile_width, self.position.y * config.tile_width), config.tile_width / 2)
 
-    def checktile(self, world):
-        tile_type = world[self.position.y][self.position.x].type
-        if tile_type == "empty":
-            ...
-        elif tile_type == "food":
-            ...
-        elif tile_type == "danger":
-            ...
+    def vision(self):
```

Comment:
Wenn du in deiner Funktion das "grid" von einem "World" Objekt verwenden willst, musst du als Parameter ein world hinzufügen.
```suggestion
    def vision(self, world):
```